iot_verifier/src/poc.rs

```diff
@@ -489,6 +494,36 @@ fn verify_denylist(pub_key: &PublicKeyBinary, deny_list: &DenyList) -> GenericVe
     //
     Ok(())
 }
+
+/// verify if gateway-gateway edge is on the deny list
+/// note that the order of the gateway keys is unimportant as edges are not considered directional
+fn verify_edge_denylist(
```

Comment:
need to add a test here to cover this ( see the existing `test_verify_denylist` test for reference )